src/raglite/_config.py

```diff
@@ -53,6 +67,12 @@ class RAGLiteConfig:
         ),
         compare=False,  # Exclude the reranker from comparison to avoid lru_cache misses.
     )
+    search_method: "SearchMethod" = field(default_factory=_default_search_method, compare=False)
+    system_prompt: str | None = None
```

Comment:
Which system prompt is this? Can we leave it out of the config? I don't believe we use one for RAG currently.

Reply:
It is the one for the specific use case. It contains information about the assistant role, language, style, etc. I think it is important to keep it here and make it available to the evaluation as it contains valuable information. Also my aim is that just by modifying the Config on could switch from one use case to another without modifying anything else. 

In other words, everything that could be modified to improve the performance should be in Config (num_chunks, search method and prompts)

Reply:
I have the feeling we'll end up with potentially many system prompts though in the future, and then it will be difficult to distinguish between them. Can we solve this with the same `partial` trick?